doc(release) add missing Release note web 23.10.4 (#3933)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

- [Dashboards] Added meta-services to metric widgets (Single metric, Metrics graph, Top/Bottom).
- [Install] Added 24.10 support for unattended script.
- [Library - view by card] Added a thumbnail that shows a snapshot of your data, taken when the dashboard is saved.

> - [Poller/Agent configurations] An interface now allows you to manage the configuration of agents, including Centreon Monitoring Agent and Telegraf.

Reviewer: ```suggestion
- [Poller/Agent configurations] You can now manage the configuration of agents through an interface. This includes Centreon Monitoring Agent and Telegraf.
```

> - [Poller/Agent configurations] A new feature is available to manage agents configuration, including Centreon Monitoring Agent and Telegraf to manage the configuration of agents.

Reviewer: ```suggestion
- [Poller/Agent configurations] An interface now allows you to manage the configuration of agents, including Centreon Monitoring Agent and Telegraf.
```
